src/Migration/AbstractMigration.php

```diff
@@ -62,6 +62,8 @@ protected function markMigrated(WC_Data $object): void
         }
 
         $object->update_meta_data($migratedKey, array_merge($executedMigrations, [$this->getVersion()]));
+
+        $object->save_meta_data();
```

Comment:
niet hierin, nu gebeurt het dubbel bij orders, plus het is niet duidelijk dat het gebeurt in deze functie. als je dit per se hier wil doen moet je ook de orders migratie aanpassen, en als je dat toch al gaat doen heb ik eigenlijk ook liever dat je meteen de plekken waar `WC_Order->save()` gedaan wordt ook meeneemnt.